pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -4111,6 +4104,68 @@ func (r *ClickHouseReader) GetLatencyMetricMetadata(ctx context.Context, metricN
 	}, nil
 }
 
+func (r *ClickHouseReader) GetMetricMetadata(ctx context.Context, metricName, serviceName string) (*v3.MetricMetadataResponse, error) {
```

Comment:
this method is expected to work only for the APM data as it contains `serviceName`? What's the planned scope of using metric metadata?

Reply:
The current scope is APM, since it uses the delta/cumulative to prepare an appropriate formula for apdex.

Reply:
ok